src/docs/product/performance/filters-display.mdx

```diff
@@ -44,7 +44,7 @@ When you select "All Transactions", the table includes all frontend and backend
 
 ### Mobile
 
-"Mobile" puts an emphasis on Mobile Vitals. Note that if you've selected a non-mobile project, this mode also displays all of those transactions.
+"Mobile" puts an emphasis on [Mobile Vitals](/product/performance/mobile-vitals/). Note that if you've selected a non-mobile project, this mode also displays all of those transactions.
```

Comment:
Should the screenshot also be added here for parity with most of the other display options?

Reply:
That makes sense. Will do